packages/terra-popup/src/_PopupOverlay.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './PopupOverlay.scss';
+
+const propTypes = {
+  /**
+   * The string representation of the index.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: [],
+};
+
+class PopupOverlay extends React.Component {
+
+  componentWillMount() {
```

Comment:
I have read a few different places that using componentDidMount is preferable over componentWillMount. What issue did you see when using componentDidMount?

There has also been some discussion about deprecating componentWillMount 
https://github.com/facebook/react/issues/7671